projects/models.py

```diff
@@ -101,14 +105,30 @@ def __str__(self):
         return self.title
 
 
+def add_project_controls(instance: Project):
+    try:
+        controls = apps.get_model("catalogs", "Controls")
+        catalog_controls = controls.objects.filter(catalog_id=instance.catalog)
+        print(catalog_controls)
+        if catalog_controls:
+            instance.controls.set(
+                catalog_controls,
+                through_defaults={"status": ProjectControl.Status.NOT_STARTED},
+            )
+    except ObjectDoesNotExist as e:
```

Comment:
I'm not sure how this exception will be raised in this case. Controls are filtered, and then before attempting to set controls, the QuerySet is checked to make sure that it's not empty. Since the input controls to `set` are coming from the DB itself, I'm not sure how the objects won't exist  in the DB.